include/libopencm3/stm32/f4/rcc.h

```diff
@@ -681,7 +690,7 @@
 
 /* RCC_PLLSAICFGR[27:24]: PLLSAIQ */
 #define RCC_PLLSAICFGR_PLLSAIQ_SHIFT		24
-#define RCC_PLLSAICFGR_PLLSAIQ_MASK		0xF
+#define RCC_PLLSAICFGR_PLLSAIQ_MASK		0xf
```

Comment:
pretty gratuitous :) you didn't change other caps to lowercase :)

Reply:
yeah, meld complained :)